src/Tickets/Flexible_Tickets/template-tags.php

```diff
@@ -165,4 +165,58 @@ function tec_tickets_get_default_ticket_type_label_lowercase( string $context =
 			$context
 		);
 	}
-}
\ No newline at end of file
+}
+
+if ( ! function_exists( 'tec_tickets_get_default_ticket_type_label_plural' ) ) {
+	/**
+	 * Returns the filtered default Ticket Type label in plural.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $context The context in which this string is filtered, e.g. 'verb' or 'template.php'.
+	 *
+	 * @return string The plural version of default ticket type label.
+	 */
+	function tec_tickets_get_default_ticket_type_label_plural( string $context = '' ): string {
```

Comment:
Question: should this be here or in the more general ET template tags? Is this exclusive of the FT feature?

Reply:
Good call, [moved](https://github.com/the-events-calendar/event-tickets/pull/2875/commits/f27d4f7224a461dd472563f05eb5383bcfaeaa27) them to base template tags for tickets: